doc/book/admin/access_control.rst

```diff
@@ -1,270 +1,865 @@
+    box.schema.user.grant('testuser','write','space','_sequence')
+
+.. _access_control_grant_sequences_functions:
+
+Using sequence functions
+************************
+
+In this example, 'testuser' is allowed to use the ``id_seq:next()`` function with a sequence named 'id_seq':
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','read,write','sequence','id_seq')
+
+In the next example, 'testuser' is allowed to use the ``id_seq:set()`` or ``id_seq:reset()`` functions with a sequence named 'id_seq':
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','write','sequence','id_seq')
+
+
+
+.. _access_control_grant_functions:
+
+Functions
+~~~~~~~~~
+
+.. _access_control_grant_functions_create_drop:
+
+Creating and dropping functions
+*******************************
+
+In this example, 'testuser' gets privileges to create :ref:`functions <box_schema-func_create>`:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','create','function')
+    box.schema.user.grant('testuser','read,write','space','_func')
+
+To let 'testuser' drop a function, grant them the following privileges:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','drop','function')
+    box.schema.user.grant('testuser','write','space','_func')
+
+
+.. _access_control_grant_functions_execute:
+
+Executing functions
+*******************
+
+To give the ability to execute a function named 'sum', grant the following privileges:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','execute','function','sum')
+
+
+
+
+
+.. _access_control_grant_users:
+
+Users
+~~~~~
+
+In this example, 'testuser' gets privileges to create other users:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','create','user')
+    box.schema.user.grant('testuser', 'read,write', 'space', '_user')
+    box.schema.user.grant('testuser', 'write', 'space', '_priv')
+
+.. _access_control_grant_roles:
+
+Roles
+~~~~~
+
+To let 'testuser' create new roles, grant the following privileges:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','create','role')
+    box.schema.user.grant('testuser', 'read,write', 'space', '_user')
+    box.schema.user.grant('testuser', 'write', 'space', '_priv')
+
+
+.. _access_control_grant_execute_code:
+
+Executing code
+~~~~~~~~~~~~~~
+
+To let 'testuser' execute Lua code, grant the 'execute' privilege to the 'lua_eval' object:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','execute','lua_eval')
+
+Similarly, executing an arbitrary SQL expression requires the 'execute' privilege to the 'sql' object:
+
+..  code-block:: lua
+
+    box.schema.user.grant('testuser','execute','sql')
+
+
+
+
+
+.. _creating_users_and_objects_granting_privileges:
+
+Example
+~~~~~~~
+
+In the example below, the created Lua function is executed under the user ID of its
 creator, even if called by another user.
 
 First, the two spaces ('u' and 'i') are created, and a no-password user ('internal')
```

Comment:
Let's use more self-explanatory and realistic entity names than `u`, `i`, and `internal`.

Reply:
Okay :) just copy-pasted this example from old docs as is and was a bit lazy to fix this.